pleiades/vocabularies/configure.zcml

```diff
@@ -72,6 +72,22 @@
     handler=".setuphandlers.remove_old_time_periods"
     />
 
+  <genericsetup:upgradeSteps
+      profile="pleiades.vocabularies:default"
+      source="0003"
+      destination="0004">
+    <genericsetup:upgradeDepends
+      title="Install relationship types registry vocabulary"
+      description=""
+      import_steps="plone.app.registry controlpanel"
```

Comment:
We do need to re-import registry.xml so that it adds a registry record for the new field that's now part of the IPleiadesSettings interface. But we don't need to re-import the control panel.